Agents/GFAAgent/inputs/config/gfaagent-debug.json

```diff
@@ -0,0 +1,55 @@
+{
+    "ServiceSpec": {
+        "Name": "gfaagent",
+        "TaskTemplate": {
+            "ContainerSpec": {
+                "Image": "ghcr.io/cambridge-cares/gfaagent:1.0.0-SNAPSHOT",
```

Comment:
At the moment, there are two debug configuration in different folders. Please move them into one folder and include how to use them in the README

If the GFA agent cannot run outside the stack, remove the non-stack debug configuration as it is not relevant.